graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsDataLoaderRegistry.kt

```diff
@@ -144,16 +151,16 @@ open class DgsDataLoaderRegistry : DataLoaderRegistry() {
     }
 
     override fun getKeys(): Set<String> {
-        return scheduledDataLoaderRegistries.keys.plus(dataLoaderRegistry.keys)
+        return scheduledDataLoaderRegistries.keys + dataLoaderRegistry.keys
     }
 
     /**
      * This will be called [org.dataloader.DataLoader.dispatch] on each of the registered
      * [org.dataloader.DataLoader]s
      */
     override fun dispatchAll() {
-        scheduledDataLoaderRegistries.forEach {
-            it.value.dispatchAll()
+        scheduledDataLoaderRegistries.forEach { (_, value) ->
```

Comment:
I'm surprised this makes a difference in performance?

Reply:
It doesn't, I just changed it since I was touching the file anyway.